src/sentry/integrations/bitbucket_server/webhook.py

```diff
@@ -44,7 +44,13 @@ def __call__(self, organization, integration_id, event):
         except Repository.DoesNotExist:
             raise Http404()
 
-        client = repo.get_provider().get_installation(integration_id, organization.id).get_client()
+        provider = repo.get_provider()
+        try:
+            installation = provider.get_installation(integration_id, organization.id)
+        except Integration.DoesNotExist:
+            raise Http404()
```

Comment:
Can you add a test to verify the fix?

Reply:
Added some tests: `tests/sentry/integrations/bitbucket_server/test_webhook.py`.